packages/ui/src/components/molecules/Button/Button.tsx

```diff
@@ -20,63 +20,109 @@ import React, {useMemo} from 'react';
 import {StyleSheet, TouchableOpacity} from 'react-native';
 import {getCommonStyles} from '../../../utils/commons-styles';
 import {useThemeColor} from '../../../theme/ThemeContext';
-import {Text} from '../../atoms';
+import {Icon, Text} from '../../atoms';
 import {Color} from '../../../theme/themes';
 
 interface ButtonProps {
-  style?: any;
-  styleTxt?: any;
   color?: Color;
-  title: string;
-  onPress?: () => void;
   disabled?: boolean;
+  FontAwesome5?: boolean;
+  iconName?: string;
+  isNeutralBackground?: boolean;
+  onDisabledPress?: () => void;
+  onPress?: () => void;
+  style?: any;
+  styleTxt?: any;
+  title?: string;
+  width?: string;
```

Comment:
width should also accept a number
It can be a fixed width defined as a number or defined using `Dimensions.get('window').width` which is also a number